dev/io.openliberty.opentracing.2.0.spi/bnd.bnd

```diff
@@ -0,0 +1,27 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion: 1.0
+
+Bundle-Name: WebSphere OpenTracing 2.0 SPI
+Bundle-Description: WebSphere OpenTracing SPI, version 2.0
+Bundle-SymbolicName: io.openliberty.opentracing.2.0.spi
+
+Import-Package: io.openliberty.opentracing.internal.tracer;version=2.0
+
+Export-Package: io.openliberty.opentracing.internal.tracer;version=2.0
```

Comment:
If this is really an SPI then the package name should not be internal; `io.openliberty.opentracing.spi.tracer`

Reply:
The UFO does not mention this SPI. You will need to update the UFO before exposing this.

Reply:
Do we need to list all SPI, API in the UFO?  This is not a new SPI.  I believe we have this exposed as SPI since 1.0.

Reply:
Will list the SPI in the UFO.

Reply:
UFO was updated